src/cinder/Triangulate.cpp

```diff
@@ -66,15 +73,16 @@ Triangulator::Triangulator()
 void Triangulator::allocate()
 {
 	mAllocated = 0;
-	
-	TESSalloc ma;
-	memset( &ma, 0, sizeof(ma) );
-	ma.memalloc = stdAlloc;
-	ma.memfree = stdFree;
-	ma.userData = (void*)&mAllocated;
-	ma.extraVertices = 2560; // realloc not provided, allow 256 extra vertices.
-
-	mTess = shared_ptr<TESStesselator>( tessNewTess( &ma ), tessDeleteTess );
+
+	mAlloc = shared_ptr<TESSalloc>( new TESSalloc() );
```

Comment:
I assume the problem is that Triangulate has an implicitly defined destructor. That means that the compiler tries to generate the destructor in every translation unit that includes the header file. However, in most translation units, it doesn't have access to the Definition of TESSalloc which it apparently needs  for the destructor of mAlloc. 
Try defining an empty destructor in Triangulate.cpp